package.json

```diff
@@ -91,6 +91,7 @@
     "singleQuote": true
   },
   "dependencies": {
+    "@types/jest-when": "2.4.1",
```

Comment:
You probably want this in `devDependencies` instead

Reply:
Ok, i moved all dependencies in dev for jest stuff.